index.js

```diff
@@ -20,6 +20,10 @@ module.exports = function (opts, cb) {
         if (typeof cb === 'function') cb(code, sig)
     });
 
+    ps.stdin.on('error', function (e) {
+        if (!e.errno || e.errno !== 'EPIPE') { throw(e); }
```

Comment:
I am completely at a loss as to how to handle this sanely. [events.js](https://github.com/nodejs/node/blob/dccccbb/lib/events.js#L155) throws an exception when no error handler is defined, otherwise it calls all error handlers in order--I'd like the ability to detect EPIPE (and possibly other errors) and ignore it without calling other error handlers, but provide default behavior for other errors (calling registered error handlers or throw if none present). But I can't figure out a way to do this within the framework of `EventEmitter`.